src/controllers/__tests__/profileController.test.ts

```diff
@@ -56,7 +55,8 @@ const proxyUserResponse = {
   preferredLanguages: aPreferredLanguages,
   spid_email: anEmailAddress,
   token: "123hexToken",
-  version: 1 as NonNegativeInteger
+  version: 1 as NonNegativeInteger,
+  last_app_version: "UNKNOWN"
```

Comment:
`last_app_version` should never be equals to `UNKNOWN`. Or is a valid `Semver` or is missing.

Reply:
https://github.com/pagopa/io-backend/pull/903/commits/4552552ce849651c5835f3e4d4a9c62fa0338427